### PR TITLE
chore: css-modules mock

### PR DESCRIPTION
**Pull request by ncomont**
**What is the problem this PR is trying to solve?**

Our css-modules mock doesn't render class names in snapshots.

**What is the chosen solution to this problem?**

Define a new Proxy, widely inspired by `identity-obj-proxy`.

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/Loader/Loader.component.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ function Loader({ id, className, size = SIZE.default }) {
 
 	return (
 		<div id={id} aria-atomic="true" aria-busy="true" className={loaderClassNames}>
+			wehs : {size}
```
Comment (romainseb) [inline packages/components/src/Loader/Loader.component.js]: Is this wanted ?

File: packages/components/src/ListView/Items/__snapshots__/items.test.js.snap [hunk 2]
```diff
@@ -7793,7 +7793,7 @@ exports[`Items should render with provided id 1`] = `
                                 type="checkbox"
                               />
                               <span
-                                className="tc-listview-item-label"
+                                className="tc-listview-item-label tc-listview-item-label"
```
Comment (acateland) [inline packages/components/src/ListView/Items/__snapshots__/items.test.js.snap]: doesn't look like it always work fine

Comment (ncomont) [reply packages/components/src/ListView/Items/__snapshots__/items.test.js.snap]: Item's classnames is defined like that : `classNames(theme['tc-listview-item-label'], 'tc-listview-item-label');`

So yes, I think this is working as expected.

Comment (ncomont) [reply packages/components/src/ListView/Items/__snapshots__/items.test.js.snap]: As seen with @acateland, theme classes are now prefixed by `theme-`.

**Pull request merged**